buildchow/templates/defaults.yaml

```diff
@@ -0,0 +1,37 @@
+- defaults:
+    name: global
+    disabled: false
```

Comment:
Disabled = true about 90% of the time. We only enable these jobs when we need them; they're disabled until needed (or until the new Beta1x branch EXISTS.)